app/units/units.css

```diff
@@ -79,6 +85,7 @@
     color: #DD432B;
     font-size: 15px;
     text-align: right;
+    cursor: pointer; /* Added cursor pointer for clear button */
```

Comment:
Please remove all comments as it is self-explanatory :)

Reply:
Yeah that's right I have added it for your convenience. It will remove it along with the other points that you have mentioned